src/dpdkstat.c

```diff
@@ -623,9 +620,9 @@ static int dpdk_read (user_data_t *ud)
   }
 
   /* Dispatch the stats.*/
-  int count = 0, i = 0, port_num = 0;
+  int count = 0, port_num = 0;
 
-  for (; i < g_configuration->num_ports; i++) {
+  for (int i = 0; i < g_configuration->num_ports; i++) {
```

Comment:
If I see this correctly, then `num_ports` is a `uint32_t`. If so, then `i` should be of the same type to avoid "comparison between signed and unsigned" warnings.